meson.build

```diff
@@ -21,8 +21,7 @@ endif
 
 flatpak_intf_dir = get_option('flatpak-interfaces-dir')
 if flatpak_intf_dir == ''
-    flatpak_required = host_machine.system() in ['linux']
-    flatpak_dep = dependency('flatpak', version: '>= 1.5.0', required : flatpak_required)
+    flatpak_dep = dependency('flatpak', version: '>= 1.5.0', required : false)
```

Comment:
This results in an automagic dependency (and thus non-reproducible build artifacts) and it feels like it would probably be better to have a project option that enables flatpak support or can be explicitly disabled.